app/controllers/registered_applications_controller.rb

```diff
@@ -5,6 +5,7 @@ def index
 
   def show
     @app = RegisteredApplication.find(params[:id])
+    @events = @app.events.group_by(&:eventname)
```

Comment:
Doing some quick research tells me... maybe? I see a couple instances where people are using the scope within the model, and getting advice to use `group_by` instead since that returns an array to instance over.

If I use the `default_scope`, does that end up something like `default_scope group('eventname')`? Then would I still declare `@events = @app.events` and iterate over it in the `show` view similar to what I already have, or is that done for my by the scope?